content/en/sensitive_data_scanner/_index.md

```diff
@@ -119,14 +168,41 @@ You can create custom scanning rules using regex patterns to scan for sensitive
+
+To add scanning rules, perform the following steps.
+
+<div class="alert alert-warning"><strong>Note</strong>: Sensitive Data Scanner in the Agent supports only predefined scanning rules from Datadog's Scanning Rule Library. The total number of scanning rules is limited to 20.</div>
+
+1. Navigate to the [Sensitive Data Scanner in the Agent][1] configuration page.
+1. Click **Add Scanning Rule**. Alternatively, click the **Add** dropdown menu on the top right corner of the page and select **Add Scanning Rule**.
+1. In the **Add library rules to the scanning group** section, select the library rules you want to use. Use the **Filter library rules** input to search existing library rules. Next to the rule name you can find the list of predefined tags for each rule.
+1. In the **Define rule target and action** section, select the action that you want to take for the matched sensitive information. **Note**: Redaction, partial redaction, and hashing are all irreversible actions.
+    - **Redact**: Replaces all matching values with the text you specify in the **Replacement text** field.
+    - **Partially Redact**: Replaces a specified portion of all matched data. In the **Redact** section, specify the number of characters you want to redact and which part of the matched data to redact.
+    - **Hash**: Replaces all matched data with a unique identifier. The UTF-8 bytes of the match is hashed with the 64-bit fingerprint of FarmHash.
+1. Optionally, add tags you want to associate with events where the values match the specified regex pattern. Datadog recommends using `sensitive_data` and `sensitive_data_category` tags. These tags can then be used in searches, dashboards, and monitors. See [Control access to logs with sensitive data](#control-access-to-logs-with-sensitive-data) for information on how to use tags to determine who can access logs containing sensitive information. **Note**: If no tags are added when saving the rule, the Agent automatically adds the `sds_agent` tag as well as `sensitive_data` and `sensitive_data_category` tags which are predefined by the rule.
+1. Click **Save**.
+
+[1]: https://app.datadoghq.com/organization-settings/sensitive-data-scanner/configuration/agent
+[2]: https://registry.terraform.io/providers/DataDog/datadog/latest/docs/resources/sensitive_data_scanner_rule
 {{% /tab %}}
 {{< /tabs >}}
 
+
 **Notes**:
 
-- Any rules that you add or update only affect data coming into Datadog after the rule was defined.
+- Any rules that you add or update affect only data coming into Datadog after the rule was defined.
 - Sensitive Data Scanner does not affect any rules you define on the Datadog Agent directly.
-- To turn off Sensitive Data Scanner entirely, set the toggle to **off** for each Scanning Group and Scanning Rule so that they are disabled.
+- After rules are added, ensure that the toggles for your scanning groups are enabled to begin scanning.
```

Comment:
🛑 "Notes" section shouldn't be shared; it should be in the tab:

> Sensitive Data Scanner does not affect any rules you define on the Datadog Agent directly.

^ This is only true for Cloud SDS.

I'd suggest we keep original **Notes for the Cloud** tab:

> - Any rules that you add or update affect only data coming into Datadog after the rule was defined.
> - Sensitive Data Scanner does not affect any rules you define on the Datadog Agent directly.
> - To turn off Sensitive Data Scanner entirely, set the toggle to **off** for each Scanning Group and Scanning Rule so that they are disabled.

And new **Notes for the Agent** tab:

> - Any rules that you add or update affect only data coming into Datadog after the rule was defined.
> - After rules are added, ensure that the toggles for your scanning groups are enabled to begin scanning.

Reply:
I still want to break disable out into its own step, which is true for both versions, so I think it's fine to remove that note from Cloud as well.

Reply:
And I think it's good to clarify that toggles should be enabled even in the Cloud version.

Reply:
👍 Agree.